### PR TITLE
uttmm 46: toolbar and routes for cms on angular

### PR DESCRIPTION
**Pull request by dgzargo**

### CODE REVIEW

File: Angular/src/app/project-participants/project-participants.component.ts [hunk 1]
```diff
@@ -14,7 +14,7 @@ import { RequestStatus } from '../request-status-enum';
 export class ProjectParticipantsComponent implements OnInit {
 
   projectParticipationRequest = new MatTableDataSource<ProjectParticipationRequest>();
-  showActionButtons: boolean = true;
```
Comment (AlexBoichuk) [inline Angular/src/app/project-participants/project-participants.component.ts]: Why did you delete variable's type?

Comment (dgzargo) [reply Angular/src/app/project-participants/project-participants.component.ts]: type declaration was unnecessary. that action was performed by ts-lint

**Comment by AlexBoichuk** [comment]
I am concerned that this pull request has changes to the code part that I did in scope of task uttmm-34, that I am going to present on our demo 27/11 and I can't be sure the code will work properly after refactoring as previously it was already tested. 
I don't mind to get these components refactored, but the refactoring is out of scope of uttmm-46, lets discuss this before making any changes in the components that I made and tested. I apriciate your efforts, but please create a new task, specially for refactoring and extract refactoring of toolbar component in a new pull request. I would like to be posted and track any changes in the code part that I am going to present on the demo, because this is important for me.

**Comment by RishkoPavel** [comment]
You have to call my component when user clicks on **any**  "Piranha CMS" button  😐😳 
Please do it as must to bee
